multiples.js

```diff
@@ -3,10 +3,13 @@
  * @param  {Number} n Number that is to be used for calculating multiples up to that Number
  * @return {Number}
  */
-exports.sumOfAMultiple = function( n ) {
+exports.sumOfAMultiple = function (n) {
   var sum = 0;
-
-  // do your work here
-
+  for (let i = 0; i < n; i++) {
```

Comment:
Your solutions won't catch all cases, see if you can figure out why and resolve 🤔